terraform/modules/application/alarms.tf

```diff
@@ -38,6 +38,44 @@ resource "aws_cloudwatch_metric_alarm" "app_memory" {
   }
 }
 
+resource "aws_cloudwatch_metric_alarm" "app_tasks_exited" {
+  alarm_actions             = [var.sns_topic_arn]
+  alarm_name                = "${var.prefix}-app-tasks-exited"
+  comparison_operator       = "GreaterThanOrEqualToThreshold"
+  datapoints_to_alarm       = 1
+  evaluation_periods        = 1
+  metric_name               = "TriggeredRules"
+  namespace                 = "AWS/Events"
+  ok_actions                = [var.sns_topic_arn]
+  period                    = 8 * 60
+  statistic                 = "Sum"
+  threshold                 = 2 * var.app_task_desired_count
```

Comment:
After artificially causing this failure for both 2 and 4 tasks, I found that seeing double the desired count fail happened reliably in an 8 minute period.

Reply:
Might be worth adding a comment about why an 8min period in the code (in case someone looks at it in the future and is wondering why)